.lighthouse/jenkins-x/bdd/terraform.yaml.gotmpl

```diff
@@ -50,17 +46,27 @@ spec:
   serviceAccount: tekton-bot
 
   applyOnCreate: true
-  applyOnUpdate: true
-  applyOnDelete: true
   ignoreDelete: false
 
 
-  postrunScript: |-
+  postApplyScript: |-
     #!/bin/bash
     set -e
     set -x
 
     echo "Terraform is done!"
+    echo "Installing terraform"
```

Comment:
Cannot get custom image to work, so installing things here. Will change tfo-gcp docker image to work with new tf operator, so we don't have to do this in the future.